internal/ingress/annotations/backendprotocol/main.go

```diff
@@ -46,20 +46,28 @@ func NewParser(r resolver.Resolver) parser.IngressAnnotation {
 // ParseAnnotations parses the annotations contained in the ingress
 // rule used to indicate the backend protocol.
 func (a backendProtocol) Parse(ing *extensions.Ingress) (interface{}, error) {
+	klog.Infof("Parsing backend protocol annotation")
 	if ing.GetAnnotations() == nil {
 		return HTTP, nil
 	}
 
+	// Proofpoint hack to make v0.24.1 compatible with deprecated "secure-backend" annotation
```

Comment:
Please remove this change. This annotation was removed in 0.21.0 https://github.com/kubernetes/ingress-nginx/pull/3203